package.json

```diff
@@ -15,17 +15,16 @@
     "@google-cloud/vision": "^2.3.2",
     "@mux/blurhash": "^0.1.2",
     "@mux/mux-node": "^7.0.0",
-    "@mux/mux-player": "^1.6.0",
```

Comment:
this went missing in the update

Reply:
Removing for now since we're only using @mux/mux-player for types, and only in conjunction with @mux/mux-player-react usage. Removing ensures that the versions are in sync (which we want to guarantee for the aforementioned reasons). This suggests we should have a better resolution in mux-player-react to remove the need of referencing mux-player altogether.